conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilAppConfig.scala

```diff
@@ -1,7 +1,8 @@
-package tech.cryptonomic.conseil.config
+package tech.cryptonomic.conseil.api
```

Comment:
The `argParser` is built with the wrong name ("conseil"), that will show on the console prompt. it should be renamed "conseil-api" for consistency with how the program is run